src/plugins/backdropScreensaver/plugin.js

```diff
@@ -42,6 +42,7 @@ class BackdropScreensaver {
                 this.currentSlideshow.hide();
                 this.currentSlideshow = null;
             }
+            return Promise.resolve();
```

Comment:
Why?

Reply:
I expect `Promise` here https://github.com/jellyfin/jellyfin-web/blob/517b69550e5c14931c816391b1fe8f7680cb6bf6/src/libraries/screensavermanager.js#L75-L77
but cannot do much in `BackdropScreensaver`.

At the same time, `BackdropScreensaver` also controls scrollbar state (`dialogHelper` to be precise). That's why I added an extra CSS class for screensaver manager. When screensaver manager removes its own class, class from `dialogHelper` remains and hides scrollbar until dialog (slideshow) disappears.